src/index.scss

```diff
@@ -46,19 +46,31 @@ body {
 
 .modal-wrapper {
   margin-top: 5rem;
-  padding: 2rem 2.5rem;
   position: absolute;
   top: 0;
```

Comment:
I'd make it vertically centered instead of hardcoded top margin.

Reply:
changed